docs/DeveloperGuide.md

```diff
@@ -36,3 +44,64 @@
 ## Instructions for manual testing
 
 {Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+
+## Parser Component
+
+Here is a general guide of how Parse Class looks like in sequence diagram.
```

Comment:
I suppose this should be "Parser Class" instead? Considering the file is Parser.java (although the png file is Parse.png)